src/content/1.7/development/database/db.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Db class
+weight: 1
+---
+
+# Using the database with the Db class
```

Comment:
Can we specify that in `executeS`, `getRow`, `getValue` and `execute` we need the DB_PREFIX ? 
In `update`, `delete`, `insert` we don't need to specify the prefix

Reply:
done